Base/Python/slicer/tests/test_slicer_util_getNodes.py

```diff
@@ -1,11 +1,24 @@
 import unittest
 import slicer
 import slicer.util
+import vtk
 
 class SlicerUtilTest(unittest.TestCase):
     def setUp(self):
         slicer.mrmlScene.AddNode(slicer.vtkMRMLScalarVolumeNode()).SetName("Volume1")
         slicer.mrmlScene.AddNode(slicer.vtkMRMLScalarVolumeNode()).SetName("Volume2")
+        slicer.mrmlScene.AddNode(slicer.vtkMRMLScalarVolumeNode()).SetName("Volume")
+        slicer.mrmlScene.AddNode(slicer.vtkMRMLScalarVolumeNode()).SetName("Volume")
+
+    def test_getNodesMultipleNodesSharingName(self):
+
+        self.assertTrue("Volume" in slicer.util.getNodes("Volume"))
+        self.assertTrue("Volume" in slicer.util.getNodes("Volume",useLists=True))
```

Comment:
These can be written as [`self.assertIn(...)`](https://docs.python.org/2.7/library/unittest.html#unittest.TestCase.assertIn) to be easier to read and to provide better error messages on failure.